src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1585,6 +1585,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="TargetOutputs" ItemName="_ProjectReferenceTargetFrameworkPossibilities" />
     </MSBuild>
 
+    <ItemGroup>
+      <!--
+        Preserve the ItemSpec value on the _ProjectReferenceTargetFrameworkPossibilities. Because relative paths in another project
+        context would be incorrect, the MSBuild task invocation needs expands the project reference paths in the MSBuild task above.
+        This is generally OK, but if the list is copied the OriginalItemSpec can become the expanded value and cause issues correlating
```

Comment:
Maybe call out that `GetTargetFrameworks` is the target which returns full paths.